tests/test_dataset.py

```diff
@@ -0,0 +1,73 @@
+import pytest
+from datasets import Dataset
+
+from delphi.dataset.token_map import token_map
+
+
+def test_token_map():
+    tokenized_dataset = Dataset.from_dict(
+        {
+            "tokens": [
+                [0, 1, 2, 3, 4, 5, 0, 6, 7],
+                [0, 1, 2, 3, 4, 5, 0, 6, 7],
+                [0, 1, 2, 3, 4, 5, 0, 6, 7],
+            ]
+        }
+    )
+    mapping = token_map(tokenized_dataset)
+    assert mapping == {
+        0: [(0, 0), (0, 6), (1, 0), (1, 6), (2, 0), (2, 6)],
+        1: [(0, 1), (1, 1), (2, 1)],
+        2: [(0, 2), (1, 2), (2, 2)],
+        3: [(0, 3), (1, 3), (2, 3)],
+        4: [(0, 4), (1, 4), (2, 4)],
+        5: [(0, 5), (1, 5), (2, 5)],
+        6: [(0, 7), (1, 7), (2, 7)],
+        7: [(0, 8), (1, 8), (2, 8)],
+    }
+    tokenized_dataset = Dataset.from_dict(
+        {
+            "tokens": [
+                [
+                    0,
+                    1,
+                    2,
+                    3,
+                    4,
+                    5,
+                    0,
+                    6,
+                    7,
+                    0,
+                    1,
+                    2,
+                    3,
+                    4,
+                    5,
+                    0,
+                    6,
+                    7,
+                    0,
+                    1,
+                    2,
+                    3,
+                    4,
+                    5,
+                    0,
+                    6,
+                    7,
+                ],
+            ]
+        }
+    )
```

Comment:
this is bad xD
you can add a noqa comment for black and keep it at reasonable number of lines

Reply:
Hahaha yeah thank you I wasn't sure how to deal with that